source/includes/rankings.md

```diff
@@ -216,3 +217,568 @@ Header            | Description
+
+# This header turns on compression to reduce the bandwidth usage and transfer time.
+headers = {'Accept-Encoding': 'deflate, gzip'}
+
+# Python Main Code Sample
+response = requests.get(request_url,
+                        auth=(username, password),
+                        params=req_params,
+                        headers=headers,
+                        stream=True)
+
+print response.status_code
+for line in response.iter_lines():
+  # Load json object and print it out
+  json_record = json.loads(line)
+  print json_record
+```
+
+```java
+// This example uses java Unirest library http://unirest.io/java.html
+
+// Request Parameters
+store = "android";      // Could be either "android" or "itunes".
+date = "{{ date }}";    // Date in YYYY-MM-DD format.
+
+// Auth Parameters
+username = "{API_KEY}"; // Replace {API_KEY} with your own API key.
+password = "X";         // Password can be anything.
+
+// Request URL
+requestUrl = "https://api.appmonsta.com/v1/stores/" + store + "/rankings/types.json";
+
+// Java Main Code Sample
+HttpResponse response = Unirest.get(requestUrl)
+  // This header turns on compression to reduce the bandwidth usage and transfer time.
+  .header("Accept-Encoding", "deflate, gzip")
+  .basicAuth(username, password)
+  .queryString("country", $countryCode),
+  .queryString("date", $date),
+  .asString();
+
+int status = response.getStatus();
+
+BufferedReader in = new BufferedReader(new InputStreamReader(response.getRawBody()));
+String line = null;
+while((line = in.readLine()) != null) {
+  System.out.println(line);
+}
+```
+
+```php
+<?php
+// Request Parameters
+$store = "android";      // Could be either "android" or "itunes".
+$date = "{{ date }}";    // Date in YYYY-MM-DD format.
+
+// Auth Parameters
+$username = "{API_KEY}"; // Replace {API_KEY} with your own API key.
+$password = "X";         // Password can be anything.
+
+// Request URL
+$url = "https://api.appmonsta.com/v1/stores/$store/rankings/types.json?date=$date";
+
+// PHP Main Code Sample
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL,$url);
+curl_setopt($ch, CURLOPT_USERPWD, $username . ":" . $password);
+curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 500);
+curl_setopt($ch, CURLOPT_WRITEFUNCTION, function($curl, $data) {
+    // Load json object and print it out
+    $json_record = json_decode((string)$data, true);
+    echo json_encode($json_record);
+    return strlen($data);
+});
+curl_exec($ch);
+curl_close($ch);
+?>
+```
+
+> The above code loads one JSON per line and prints it out:
+
+```json
+{"rank_id":"topfreeapplications","name":"Top Free"}
+{"rank_id":"newapplications","name":"New Applications"}
+{"rank_id":"newgameswelove","name":"New Games We Love RSS"}
+
+{"rank_id":"apps_movers_shakers","name":"Trending Apps"}
+{"rank_id":"apps_tablet_featured","name":"Staff Picks for Tablet"}
+{"rank_id":"apps_topselling_new_free","name":"Top New Free"}
+```
+
+Request all ranking types by date.
```

Comment:
OK, we need to really explain what this means.

Reply:
Ranking types are ranking list type (names) as defined by the Google Play or iTunes store. If you use (aggregated ranks API call link), then you can use this API call to map ranking types to their human-readable names.